### DISCUSSION TITLE
Feature request - support connection to Hive Metastore

### DISCUSSION THREAD

**Discussion by prmoore77** [duckdb/duckdb#12940] [Ideas]
It would be cool if DuckDB allowed one to connect to a Hive Metastore and leverage the metadata for table -> file mapping.

Other metastores - such as Unity Catalog, AWS Glue, and more would also be great.

This would allow folks to easily use DuckDB to query their data lakes - without having to duplicate schema and file metadata mapping in a DuckDB database persistent file.

Thanks!

**Comment by prmoore77** [comment]
cc @pdet and @ianmcook

**Comment by aldencabajar** [comment]
Any current activity towards this feature? Would be really cool to be able to do this!

**Comment by prmoore77** [comment]
Hi @aldencabajar - it looks like today's announcement of: "DuckLake" addresses this requirement very well: https://ducklake.select

**Reply by aldencabajar** [reply]
@prmoore77 this looks quite promising. Thanks for the info!
